homeassistant/components/fibaro/event.py

```diff
@@ -0,0 +1,68 @@
+"""Support for Fibaro event entities."""
+from __future__ import annotations
+
+from pyfibaro.fibaro_device import DeviceModel, SceneEvent
+from pyfibaro.fibaro_state_resolver import FibaroEvent
+
+from homeassistant.components.event import (
+    ENTITY_ID_FORMAT,
+    EventDeviceClass,
+    EventEntity,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import FibaroController, FibaroDevice
+from .const import DOMAIN
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up the Fibaro event entities."""
+    controller: FibaroController = hass.data[DOMAIN][entry.entry_id]
+
+    entities = []
+    for device in controller.fibaro_devices[Platform.EVENT]:
+        for scene_event in device.central_scene_event:
+            # Each scene event represents a button on a device
+            entities.append(FibaroEventEntity(device, scene_event))
+
+    async_add_entities(entities, True)
+
+
+class FibaroEventEntity(FibaroDevice, EventEntity):
+    """Representation of a Fibaro Event Entity."""
+
+    def __init__(self, fibaro_device: DeviceModel, scene_event: SceneEvent) -> None:
+        """Initialize the Fibaro device."""
+        super().__init__(fibaro_device)
+        self.entity_id = ENTITY_ID_FORMAT.format(
+            f"{self.ha_id}_button{scene_event.key_id}"
```

Comment:
Seems like a tiny tweak
```suggestion
            f"{self.ha_id}_button_{scene_event.key_id}"
```

Reply:
Changed as suggested.